admin/assets/javascripts/details.polyfill.js

```diff
@@ -9,6 +9,11 @@
 ;(function () {
   'use strict'
 
+  // Prevent using this polyfill on new toolkit pages as the functionality is already provided
+  if (document.getElementsByClassName('govuk-template').length === 1) {
```

Comment:
can we guarantee that there will only ever be one item with this class name?

Reply:
it's a unique class that's on the very top of the newer toolkit's layout https://design-system.service.gov.uk/styles/page-template/